pkg/machine/qemu/config.go

```diff
@@ -35,6 +43,22 @@ type MachineVM struct {
 	RemoteUsername string
 	// Whether this machine should run in a rootful or rootless manner
 	Rootful bool
+	// UID is the numerical id of the user that called machine
+	UID int
+}
+
+// GetUID is helper function to get the userid that should be
```

Comment:
Should we have a GetGID?

Reply:
i dont think we need it r/n, but we can in the future